moveit_kinematics/ikfast_kinematics_plugin/templates/ikfast61_moveit_plugin_template.cpp

```diff
@@ -544,12 +544,20 @@ int IKFastKinematicsPlugin::solve(KDL::Frame& pose_frame, const std::vector<doub
       return solutions.GetNumSolutions();
 
     case IKP_TranslationXAxisAngle4D:
```

Comment:
Please add a comment from the latest ikfast documentation, found here: http://openrave.org/docs/latest_stable/openravepy/ikfast/

> * TranslationXAxisAngle4D, TranslationYAxisAngle4D, TranslationZAxisAngle4D - end effector origin reaches desired 3D translation, manipulator direction makes a specific angle with x/y/z-axis (defined in the manipulator base link’s coordinate system)

Reply:
Done.